l3build-file-functions.lua

```diff
@@ -368,10 +367,10 @@ end
 
 -- Rename
 function ren(dir, source, dest)
-  local dir = dir .. "/"
+  dir = dir .. "/"
   if os_type == "windows" then
-    local source = gsub(source, "^%.+/", "")
-    local dest = gsub(dest, "^%.+/", "")
```

Comment:
both are already local
BTW, there is a problem with the '+' in the pattern.
Either it is required, and we are allowed to refer to the parent directory, or useless.